src/app/_components/CheckBox/CheckBox.js

```diff
@@ -2,9 +2,10 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import TickIcon from '@icons/tickIcon.svg';
+// import { TickIcon } from '../Icons';
```

Comment:
Please remove this line if you've changed the approach TickIcon is used. Also, - update `<TickIcon/>` usage below if it it just an .svg.

Reply:
Removed the commented line. The usage was also updated. There must be classes added there to account for multiple checkbox states. Styles are updated and working as expected